Comment-Code-Poltergeists/nutshell
# Gc rb layout

# Description

Setup the boilerplate HTML/CSS to show the different containers for articles/events/to-do/friends/messages/ads 😆 

Fixes #37

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality/styling)

# Testing Instructions

`git fetch --all`
`git checkout gc-rb-layout`
Make sure your initial HTML page loads with the pink/grey divs showing all the parent elements for the fullscreen view

# Checklist:
